src/main/java/org/apache/sling/feature/cpconverter/handlers/slinginitialcontent/BundleSlingInitialContentExtractContext.java

```diff
@@ -117,4 +120,13 @@ public JcrNamespaceRegistry getNamespaceRegistry() {
     public List<PathEntry> getPathEntryList() {
         return new ArrayList<>(pathEntryList);
     }
+
+    @NotNull
+    public boolean isSlingNodeTypesEntry(@NotNull JarEntry jarEntry) {
+        if(slingNodeTypes == null){
+            return false;
+        }
+        
+        return jarEntry.getName().equals(slingNodeTypes);
```

Comment:
This is potentially a comma separated list. Also a unit test/IT covering this would be good.

Reply:
Again a good point.. implemented this and pushed.